x-pack/test/functional/apps/transform/creation_runtime_mappings.ts

```diff
@@ -0,0 +1,373 @@
+              await transform.wizard.assertGroupByInputExists();
+              await transform.wizard.assertGroupByInputValue([]);
+              await transform.wizard.addGroupByEntry(
+                index,
+                entry.identifier,
+                entry.label,
+                entry.intervalLabel
+              );
+            }
+
+            await transform.testExecution.logTestStep('adds the aggregation entries');
+            await transform.wizard.addAggregationEntries(testData.aggregationEntries);
+
+            await transform.testExecution.logTestStep('displays the advanced pivot editor switch');
+            await transform.wizard.assertAdvancedPivotEditorSwitchExists();
+            await transform.wizard.assertAdvancedPivotEditorSwitchCheckState(false);
+
+            await transform.testExecution.logTestStep('displays the advanced configuration');
+            await transform.wizard.enableAdvancedPivotEditor();
+            await transform.wizard.assertAdvancedPivotEditorContent(
+              testData.expected.pivotAdvancedEditorValueArr
+            );
+          }
+
+          if (isLatestTransformTestData(testData)) {
+            await transform.testExecution.logTestStep('sets latest transform method');
+            await transform.wizard.selectTransformFunction('latest');
+            await transform.testExecution.logTestStep('adds unique keys');
+            for (const { identifier, label } of testData.uniqueKeys) {
+              await transform.wizard.assertUniqueKeysInputExists();
+              await transform.wizard.assertUniqueKeysInputValue([]);
+              await transform.wizard.addUniqueKeyEntry(identifier, label);
+            }
+            await transform.testExecution.logTestStep('sets the sort field');
+            await transform.wizard.assertSortFieldInputExists();
+            await transform.wizard.assertSortFieldInputValue('');
+            await transform.wizard.setSortFieldValue(
+              testData.sortField.identifier,
+              testData.sortField.label
+            );
+          }
+
+          await transform.testExecution.logTestStep('loads the transform preview');
+          await transform.wizard.assertPivotPreviewLoaded();
+
+          await transform.testExecution.logTestStep('shows the transform preview');
+          await transform.wizard.assertPivotPreviewChartHistogramButtonMissing();
+          for (const { column } of testData.expected.previewData) {
+            await transform.wizard.assertPivotPreviewColumnValuesNotEmpty(column);
+          }
+
+          // cell virtualization means the last column is cutoff in the functional tests
+          // https://github.com/elastic/eui/issues/4470
+          // await transform.wizard.assertPivotPreviewColumnValues(
+          //   testData.expected.transformPreview.column,
+          //   testData.expected.transformPreview.values
+          // );
```

Comment:
shall we delete it then?

Reply:
I think I'm going to keep this here for now